tests/x11regressions/empathy/empathy_irc.pm

```diff
@@ -48,16 +48,13 @@ sub run() {
     };
 
     # join openqa channel
-    if (sle_version_at_least('12-SP2')) {
+    send_key "ctrl-j";
+    if (!check_screen 'empathy-join-room', 4) {
```

Comment:
No because it is executed via an already running application (empathy).

The point is that you can use assert screen with multiple tags if you are at a state/needle A and you want to go to and check state B or state C.

But can you do the same if you are at a state A and want to check state B? In this case you have a race condition where a needle for state A might match before openQA has time to go to state B.